upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -334,16 +334,16 @@ function verify_mmhealth_summary() {
 	while IFS= read -r line
 	do
 		comp=$(echo $line|awk '{print $1}')
-		failed=$(echo $line|awk '{print $3}')
-		degraded=$(echo $line|awk '{print $4}')
-		if [ ${failed} -ne 0 ]; then
-			print error "${CHECK_FAIL} ${failed} failed $comp found."
-			unhealthy=1
-		fi
-		if [ ${degraded} -ne 0 ]; then
+                failed=$(echo $line|awk '{print $3}')
+                degraded=$(echo $line|awk '{print $4}')
+                if [ "$failed" = "0" ]; then
```

Comment:
Need -eq instead of = , right. Same for other place as well.

Reply:
good catch.. it should have been !=